packages/react-components/src/theming/FluentV9ThemeProvider.tsx

```diff
@@ -54,5 +54,5 @@ export const FluentV9ThemeProvider = (props: FluentV9ThemeProviderProps): JSX.El
 const FluentProviderWithStylesOverrides: React.FC<FluentProviderProps> = (props) => {
   const classes = useFluentV9Wrapper();
 
-  return <FluentProvider {...props} className={classes.body} />;
+  return <FluentProvider {...props} className={classes.body} applyStylesToPortals={false} />;
```

Comment:
Does this break the image overlay, or any other areas of the code?

Reply:
From storybook it looks fine

Reply:
Why do we need to disable it?

https://fluentuipr.z22.web.core.windows.net/pull/27241/public-docsite-v9/storybook/index.html?path=/docs/components-fluentprovider--default#apply-styles-to-portals